### ISSUE TITLE
New: I Am Christmas

### ISSUE THREAD

**Issue by tomkeays** [syracuse-singers/xmas#18]
A modern English carol by John Conolly and Bill Meeks. This one will require transcription.

**Comment by tomkeays** [comment]
I Am Christmas
~ Bill Meek, John Conolly

I will sew a braid of gold
On gray December's ragged sleeve,
Teach the crabbed and jaded soul
How to give, how to receive;
For rooms are thick with magic now,
The tree its soft light throwing;
The mistletoe, the holly bough
My age-old spell bestowing.

I am warmth and I am light
And I am kith and kin,
A candle in your longest night.
I am Christmas. Let me in.
I am Christmas. Let me in.

For I bring stories by the hearth,
Delight in half-forgotten names,
Apple logs on fragrant fires
With flick'ring faces in the flames.
As the year draws in its days
And tired leaves are falling,
I will brighten darkened ways
Where dusk is early calling.

I am warmth and I am light
And I am kith and kin,
A candle in your longest night.
I am Christmas. Let me in.
I am Christmas. Let me in.

I can take the weary miles
And weave a carpet to your door,
Guide the dusty wand'rers home
Safely to your side once more.
I can cheer the bitter days
With tunes to set you singing.
My standard in your heart I'll raise,
Joy and comfort bringing.

I am warmth and I am light
And I am kith and kin,
A candle in your longest night.
I am Christmas. Let me in.
I am Christmas. Let me in.

I bring churches all aglow
And carols on the midnight air,
Colored windows streaked with snow
That gild the congregations there;
For young and old shall join and sing
To mark the longest turning.
From one glad candle that I bring,
Ten thousand more are burning.

I am warmth and I am light
And I am kith and kin,
A candle in your longest night.
I am Christmas. Let me in.
I am Christmas. Let me in.


https://mudcat.org/thread.cfm?threadid=134556


From Finest Kind's 2018 CD, "I Am Christmas": "I Am Christmas, by Lincolnshire's Bill Meek and John Conolly, was learned from Keith Kendrick, and captures the warmth and light of the Christmas season perfectly."


John Conolly - I Am Christmas
https://www.youtube.com/watch?v=ETCZR5HySkQ

**Comment by tomkeays** [comment]
FWIW, chord suggestions from the Mudcat thread...

A       E      D       A       D       A       Bm      E
I will sew a braid of gold on grey December's ragged sleeve – 
A          E         D    A    D      A         E      A
Teach the crabbed and jaded soul how to give – how to receive..
    E                   A             D       A       
For rooms are thick with magic now– the Tree its soft light
    E
throwing. . 
    A               D    A         A       A       E   A
The mistletoe – the holly-bough – my age-old spell bestowing . . 

I bring stories by the hearth - delight in half-forgotten names –
Apple-logs on fragrant fires, with flickering faces in the flames . . 
As the year draws in its days and tired leaves are falling –
I can brighten darkened ways, where dusk is early calling . . .


             E          D    A    
CHORUS: I am warmth and I am light
                   D    A       E   
                And I am Kith and Kin . . .
                   A             D       A
                A candle in your longest night –
                D    A          E
                I am Christmas – let me in . . .
                D    A          E      A
                I am Christmas – let me in.

**Comment by tomkeays** [comment]
Still need to get the group to vet this. Since I set it by ear, there's doubtless a lot of wonkiness to work out. 

https://syracuse-singers.github.io/xmas/pdf/i_am_christmas.pdf

**Comment by tomkeays** [comment]
John Conolly gave permission for our caroling group to sing the song as long as we provided a "Copyright by Bill Meek and John Conolly" notice on the transcription. He also put me in touch with [Graham Pratt](http://www.grahamandeileenpratt.co.uk/), who leads the Sheffield Folk Chorale to see if he could provide an arrangement. Graham also gave us permission to use his arrangement (he says he plans to publish it in the fifth volume of his books). There is a [video of them singing the song](https://www.youtube.com/watch?v=UGBARewAeMc) which, while not exactly the same as the version by Finest Kind that inspired me to add this song to our repertoire, is certainly a good starting point.

**Comment by tomkeays** [comment]
At practice, because Graham Pratt's choral arrangement splits the melody between parts in a way that we're not entirely on board with, we're going to table singing this one out for this year.

The idea we discussed was doing a new arrangement where the melody, as John Conolly wrote it and/or how Finest Kind do it, is transcribed straight for the soprano part. What changes are required for the other parts will have to be determined.

As an interim measure, I am going to combine the soprano/alto parts and the tenor/bass parts of Pratt's choral arrangement to share a common staff, as we do in other song arrangements; this, provided that moving the tenor part from the tenor to the bass clef is not problematic. 

For reference for restructuring the arrangement, here are two videos. 

- John Conolly (solo): https://www.youtube.com/watch?v=ETCZR5HySkQ
- Sheffield Folk Chorale (4-part): https://www.youtube.com/watch?v=UGBARewAeMc

**Comment by tomkeays** [comment]
So, who knew. Transcription from Nowell Sing We Clear

![I Am Christmas 01](https://user-images.githubusercontent.com/171037/71050317-e39bfc00-2112-11ea-9e93-aff203cc554d.jpg)

![I Am Christmas 02](https://user-images.githubusercontent.com/171037/71050332-eb5ba080-2112-11ea-9b47-044ba9a916fb.jpg)

**Comment by tomkeays** [comment]
I am using the NSWC harmony for the verses... mostly. I did change the melody line a bit to more closely match Finest Kind's version and I fiddled with the phrasing is a few places. For the harmony in the chorus, I kept NSWC's soprano and tenor parts and added Graham Pratt's alto and bass parts, although I moved the latter closer to NSWC's phrasing. 

https://github.com/syracuse-singers/xmas/blob/master/pdf/i-am-christmas__harmony.pdf
